src/Umbraco.Web.BackOffice/Controllers/MemberGroupController.cs

```diff
@@ -26,45 +28,62 @@ public class MemberGroupController : UmbracoAuthorizedJsonController
         private readonly IMemberGroupService _memberGroupService;
         private readonly UmbracoMapper _umbracoMapper;
         private readonly ILocalizedTextService _localizedTextService;
+        private readonly RoleManager<IdentityRole> _roleManager;
 
         public MemberGroupController(
             IMemberGroupService memberGroupService,
             UmbracoMapper umbracoMapper,
-            ILocalizedTextService localizedTextService
+            ILocalizedTextService localizedTextService,
+            RoleManager<IdentityRole> roleManager
             )
         {
             _memberGroupService = memberGroupService ?? throw new ArgumentNullException(nameof(memberGroupService));
+            _roleManager = roleManager ?? throw new ArgumentNullException(nameof(roleManager));
             _umbracoMapper = umbracoMapper ?? throw new ArgumentNullException(nameof(umbracoMapper));
-            _localizedTextService =
-                localizedTextService ?? throw new ArgumentNullException(nameof(localizedTextService));
+            _localizedTextService = localizedTextService ?? throw new ArgumentNullException(nameof(localizedTextService));
         }
 
+        //TODO: are there any repercussions elsewhere for us changing these to async?
+
         /// <summary>
         /// Gets the member group json for the member group id
         /// </summary>
         /// <param name="id"></param>
         /// <returns></returns>
-        public ActionResult<MemberGroupDisplay> GetById(int id)
+        public async Task<ActionResult<MemberGroupDisplay>> GetById(int id)
         {
-            var memberGroup = _memberGroupService.GetById(id);
+            //TODO: did we envisage this - combination of service and identity manager?
```

Comment:
Nope, we shouldn't use the role manager for these editors, only our member service. The Role manager will almost solely be used for public access.